config/zephyr/Kconfig

```diff
@@ -368,14 +370,34 @@ config CHIP_ICD_ACTIVE_MODE_DURATION
 
 config CHIP_ICD_ACTIVE_MODE_THRESHOLD
 	int "Intermittently Connected Device active mode threshold (ms)"
-	default 300
+	default 0 if !CHIP_ICD_LIT_SUPPORT
+	default 5000 if CHIP_ICD_LIT_SUPPORT
 	help
 	  Provides the Intermittently Connected Device active mode threshold in milliseconds.
 	  It determines the minimum amount of time the device shall stay in the active mode after the network activity.
 
+config CHIP_ICD_LIT_SUPPORT
+	bool "Intermittenly Connected Device Long Idle Time support"
+	imply CHIP_ICD_CHECK_IN_SUPPORT
+	imply CHIP_ICD_UAT_SUPPORT
+	help
+	  Enables the Intermittently Connected Device Long Idle Time support in Matter.
+	  It also implies the ICD Check-In and UAT features support that are mandatory for LIT device.
+
+config CHIP_ICD_CHECK_IN_SUPPORT
```

Comment:
Is it worth having options for check-in and UAT given that they are obligatory for LIT?

Reply:
Yes, they are mandatory, so I added implies for them to the LIT Kconfig. But they may be enabled for SIT as well, so I believe separate Kconfigs are useful.

Reply:
Btw they do not seem to be useful in this PR, because it's only change required for testing TE 1.3. However I'm working on adding new gn flags to conditionally cut off these features from the cluster code, so it will be good to have Kconfigs mapped to gn flags.

Reply:
Ok, I see. Btw. why is UAT short for User Active Mode Trigger? Should it be UAMT?

Reply:
Ah, got it, that's the feature name. Pretty confusing :)

Reply:
Yes, I also spotted it, but this is the shortcut used in the Matter spec, so I just followed the convention in the code.